lib/addons/p5.dom.js

```diff
@@ -2880,7 +2880,8 @@
     this.setModified(true);
     return this;
   };
-  p5.MediaElement.prototype.get = function(x, y, w, h) {
+  p5.MediaElement.prototype.get = p5.Renderer2D.prototype.get;
```

Comment:
Also I understand that the p5.Renderer2D is more or less just pointing upwards to p5.Renderer.get but is there a reason why you don't make that explicit here so it is easier to know where to go looking for the method logic?

Reply:
i did it that way because I think it's `Renderer2D`'s business to know how its `get` is implemented, not `MediaElement`'s. if, down the line, someone decides to implement `Renderer2D.get` differently, this line would still work.

Reply:
how about just adding a line to Renderer2D:
```javascript
// p5.Renderer2D.prototype.get = p5.Renderer.prototype.get;
```

commented out because it's a no-op.

Reply:
That seems like a good call!

Reply:
can we add some commented text above that line to explain why its there?

Reply:
i added the comment `// .get() is not overridden`